sample/src/sample/service.clj

```diff
@@ -219,10 +217,11 @@
         {:post add-order}
         ["/:id" ^:interceptors [load-order-from-db]
          {:get get-order-by-id}]]
+
        ["/secure" ^:interceptors [basic-auth] {:delete delete-db}]
 
        ["/doc" {:get [(swagger/swagger-doc)]}]
-       ["/*resource" {:get [(swagger/swagger-ui)]}]]]]))
+       ["/swagger-ui/*resource" {:get [(swagger/swagger-ui)]}]]]]))
```

Comment:
it's a feature - you can use the old linear-search router if you want to retain the old behavior, but the prefix tree router does not support this (pedestal/pedestal#330)